cmake/CheckSymbolsExist.cmake

```diff
@@ -0,0 +1,21 @@
+# Check if each symbol in the symbol list exists,
+# and define PREFIX__HAVE_SYMNAME to 1 if yes.
+#
+# Global variables that it uses:
+# - CHECK_SYMBOL_EXTRA_HEADERS:LIST
+
+include(CheckSymbolExists)
+
+macro(CHECK_SYMBOLS_EXIST SYMLIST PREFIX)
+  list(JOIN CHECK_SYMBOL_EXTRA_HEADERS ";" CHECK_SYMBOL_EXTRA_HEADERS_JOINED)
```

Comment:
`list(JOIN)` requires cmake 3.12, let's avoid using such recent cmake? (released in 2018-07)

P.S. fails travis-ci, although we should not care about this already https://travis-ci.org/github/libevent/libevent/jobs/663067267#L928

Reply:
Actually AFAICS this line can be simply dropped and will just work (since list delimiter is `;` anyway)

Reply:
Has been corrected